src/controller/mod.rs

```diff
@@ -13,6 +13,7 @@
 //!    app::{AppContext, Hooks},
 //!    boot::{create_app, BootResult, StartMode},
 //!    controller::AppRoutes,
+//!    environment::Environment,
```

Comment:
Why? If you go three lines down, you'll already find this declaration.

Reply:
Thank you, I have removed the duplicate and sorted them alphabetically, following the convention in other docs